CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 # Next version
 
+- Fix: New publish workflow with foundry
```

Comment:
My change should not be in your commit. 
And you should maybe add your own change log line for this change